src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -61,4 +77,51 @@ public ResponseEntity<JobModel> registerDrsAliases(List<DrsAliasModel> aliases)
     String jobId = drsService.registerDrsAliases(aliases, userReq);
     return ControllerUtils.jobToResponse(jobService.retrieveJob(jobId, userReq));
   }
+
+  @Override
+  public ResponseEntity<DatasetModel> adminRetrieveDataset(UUID id) {
+    List<DatasetRequestAccessIncludeModel> include =
+        List.of(
+            DatasetRequestAccessIncludeModel.ACCESS_INFORMATION,
+            DatasetRequestAccessIncludeModel.DATA_PROJECT,
+            DatasetRequestAccessIncludeModel.PROFILE,
+            DatasetRequestAccessIncludeModel.PROPERTIES,
+            DatasetRequestAccessIncludeModel.SCHEMA,
+            DatasetRequestAccessIncludeModel.STORAGE);
```

Comment:
If the intent here is to include all members, it's better to use `values()` rather than listing each enum member, otherwise any new members would have to be added manually
```suggestion
        List.of(DatasetRequestAccessIncludeModel.values())
```

Reply:
There is a DatasetRequestAccessIncludeModel.NONE value. If we use .values(), this NONE value will be included. Will this cause problems?

Reply:
> If we use .values(), this NONE value will be included. Will this cause problems?

This rings a bell for me which is why I think I added all of them individually.